### PR TITLE
doc/rbd/rbd-cloudstack.rst: include support for multiple monitors

### PR DESCRIPTION
**Pull request by zap51**
Edit doc/rbd/rbd-cloudstack.rst to include support for multiple monitors.

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: doc/rbd/rbd-cloudstack.rst [hunk 1]
```diff
@@ -117,7 +117,7 @@ To add a Ceph block device as Primary Storage, the steps include:
    
 #. Add cluster information (``cephx`` is supported).
 
-   - For **RADOS Monitor**, provide the IP address of a Ceph monitor node.
+   - For **RADOS Monitor**, provide the IP addresses, or DNS names of Ceph monitors. Please note that support for comma-separated list of IP addresses is only present in Cloudstack v4.18.0.0 or later.
```
Comment (idryomov) [inline doc/rbd/rbd-cloudstack.rst:120]:
```suggestion
   - For **RADOS Monitor**, provide the IP addresses or DNS names of Ceph monitors. Please note that support for comma-separated list of IP addresses or DNS names is only present in Cloudstack v4.18.0.0 or later.
```

**Comment by anthonyeleven** [comment]
Didn't we just cover this in another PR?

**Comment by idryomov** [comment]
> Didn't we just cover this in another PR?

It got closed due to issues with rebasing.  The suggested change above is carried over from there.

**Comment by anthonyeleven** [comment]
Ugh.  Rebasing!  I hate rebasing!  Even with strawberries!

**Review by anthonyeleven — APPROVED** [review]

**Comment by zdover23** [comment]
> Ugh. Rebasing! I hate rebasing! Even with strawberries!

He just turned 97 last week.

**Comment by anthonyeleven** [comment]
Spaceballs! The Merge Request!

**Comment by anthonyeleven** [comment]
Crap, I hit the commit button on Ilya's suggestion which has bolloxed the signed-off-by check.  Zac, is there a workaround?

**Comment by zdover23** [comment]
@zap51: Squash the two commits in this PR into a single commit, as described here: https://docs.ceph.com/en/quincy/start/documenting-ceph/#squash-extraneous-commits

If this is confusing, you can contact me on Slack and I will talk you through it.

**Comment by zdover23** [comment]
> Crap, I hit the commit button on Ilya's suggestion which has bolloxed the signed-off-by check. Zac, is there a workaround?

I think @zap51 can check out this feature branch and then run "git pull", which will pull in the commit that you made. Then @zap51 can squash your commit into his. However, the act of squashing is daunting and hard to learn, like a good golf swing. So: @zap51: run "git pull" from within the "patch-1" branch of your local working copy, and then follow the squashing procedure that I linked to in https://github.com/ceph/ceph/pull/52607#issuecomment-1648583315.

If that doesn't work, we can just close this and make a third pull request. I'll approve it immediately and merge it as soon as it has passed the checks (probably about fourteen minutes after it is raised), and I'll backport it.

**Comment by anthonyeleven** [comment]
Promise I'll keep my fat fingers away

**Comment by idryomov** [comment]
@zap51 opened a new PR because they didn't want to bother with rebasing.  Since "we" screwed it up here, I went ahead and fixed it.

**Review by idryomov — APPROVED** [review]

**Comment by anthonyeleven** [comment]
I am in your debt, good sir.

**Comment by zdover23** [comment]
> @zap51 opened a new PR because they didn't want to bother with rebasing. Since "we" screwed it up here, I went ahead and fixed it.

You must teach me of this strange power you have.

**Comment by idryomov** [comment]
> You must teach me of this strange power you have.

Hah, it's just something that GitHub allows:

> Maintainers are allowed to edit this pull request. 

See https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork.  @zap51 enabled it on this PR so I was able to amend the commits.

**Comment by zap51** [comment]
Good Sirs, @anthonyeleven @idryomov @zdover23 I'm in your debt. I really appreciate the way you guide and appreciate. Really thankful, I'll come back with more contributions and hopefully learn git before that!

**Pull request merged**